Documentation/k8s-object.md

```diff
@@ -12,25 +12,56 @@ Object storage exposes an S3 API to the storage cluster for applications to put
 
 This guide assumes you have created a Rook cluster as explained in the main [Kubernetes guide](kubernetes.md)
 
-## Rook Client
+## Create the Object Store
 
-Setting up the object storage requires running `rookctl` commands with the [Rook toolbox](kubernetes.md#tools) pod. This will be simplified in the future with a CRD for the object stores.
+Now we will create the object store, which starts the RGW service in the cluster with the S3 API.
+Specify your desired settings for the object store in the `rook-object.yaml`. For more details on the settings see the [Object Store CRD](object-store-crd.md).
 
-## Create the Object Store and User
+```yaml
+apiVersion: rook.io/v1alpha1
+kind: ObjectStore
+metadata:
+  name: my-store
+  namespace: rook
+spec:
+  metadataPool:
+    replicated:
+      size: 3
+  dataPool:
+    erasureCoded:
+      dataChunks: 6
+      codingChunks: 2
+  gateway:
+    type: s3
+    sslCertificateRef: 
+    port: 80
+    securePort:
+    instances: 1
+    allNodes: false
+```
 
-Now we will create the object store, which starts the RGW service in the cluster with the S3 API.
-From within the rook-tools container, run the following:
+If you are using a version of Kubernetes earlier than 1.7, you will need to slightly modify one setting to be compatible with TPRs (deprecated in 1.7). Notice the different casing.
```

Comment:
this should probably have a header to call it out better

Reply:
yes, i'll do that thanks